input-explainer.md

```diff
@@ -280,6 +280,91 @@ When the canvas receives a `pointerdown` event an `XRInputSource` is created wit
 
 For each of these events the `XRInputSource`'s target ray must be updated to originate at the point that was interacted with on the canvas, projected onto the near clipping plane (defined by the `depthNear` attribute of the `XRSession`) and extending out into the scene along that projected vector.
 
+## Controller State
+
+Some applications need more than point-and-click style interaction provided by the `select` events. For devices that make use of controllers, more complete information about the state of those controller's inputs (buttons, triggers, touchpads, etc) can be observed via the `XRInputSource`'s `controller` attribute.
+
+The `controller` attribute is an instance of the `XRTrackedController` interface if the `XRInputState` represents an XR controller device with a subset of common inputs, or `null` otherwise. Examples of controllers that may expose their state this way include Oculus Touch, Vive wands, Oculus Go and Daydream controllers, or other similar devices. More traditional gamepads, such as XBox or Playstation controllers, should not be exposed using this interface, nor should tracked devices without discreet inputs, such as optical hand tracking. The intent of the `XRTrackedController` interface is not to capture the complete range of inputs for all possible current and future XR controllers, but instead to expose the most commonly observed inputs in an easy-to-consume fashion.
```

Comment:
typo? discreet -> discrete

Reply:
Microsoft uses `Xbox` capitalization in [its materials](https://account.microsoft.com/account/xboxlive) vs. `XBox`

Reply:
[Sony uses](https://www.sony.com/electronics/playstation) `PlayStation` vs `Playstation`